go.mod

```diff
@@ -70,6 +70,7 @@ require (
 	github.com/containerd/cgroups v1.1.0 // indirect
 	github.com/containerd/ttrpc v1.2.1 // indirect
 	github.com/containerd/typeurl/v2 v2.1.0 // indirect
+	github.com/containers/ocicrypt v1.1.7
```

Comment:
We don't usually claim an intended module in this require clause. Could you relocate it to the first require clause?

Reply:
done